src/components/login-view/login-view.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react';
+import Form from 'react-bootstrap/Form';
+import Button from 'react-bootstrap/Button';
+
+import './login-view.scss';
+
+export function LoginView(props) {
+	const [username, setUsername] = useState('');
+	const [password, setPassword] = useState('');
+
+	const handleSubmit = (e) => {
+		e.preventDefault();
+		console.log(username, password);
+		/* Send a request to the server for authentication */
+		/* then call props.onLoggedIn(username) */
+		props.onLoggedIn(username);
+	};
+
+	return (
+		<Form>
+			<h3>Please Login</h3>
+			<Form.Group>
+				<Form.Label>Username</Form.Label>
+				<Form.Control
+					type="text"
+					placeholder="Enter username"
+					value={username}
+					onChange={(e) => setUsername(e.target.value)}
+				/>
+			</Form.Group>
+
+			<Form.Group>
+				<Form.Label>Password</Form.Label>
+				<Form.Control
+					type="password"
+					placeholder="Password"
+					value={password}
+					onChange={(e) => setPassword(e.target.value)}
+				/>
+			</Form.Group>
+			<Button variant="primary" type="button" onClick={handleSubmit}>
+				Submit
+			</Button>
```

Comment:
You'd probably want to add in a button to register so it will show the registration page.

```
		<Button variant="primary" type="button" onClick={props.onRegistrationClick}>
				Register
			</Button>
```
This should do the trick.

Reply:
Your suggestion makes so much sense :D Thank you so much.

I implemented your suggestion and the app is working as required.

I'm still connecting the dots with React, and I have a whole playlist to go through, but I think I'm getting the hang of it thanks to your feedback and my mentor's feedback.

Thanks again.